x-pack/plugins/fleet/server/services/package_policy.test.ts

```diff
@@ -118,6 +118,75 @@ describe('Package policy service', () => {
       ]);
     });
 
+    it('should work with input condition', async () => {
+      const inputs = await packagePolicyService.compilePackagePolicyInputs(
+        ({
+          data_streams: [
+            {
+              type: 'logs',
+              dataset: 'package.dataset1',
+              streams: [{ input: 'log', template_path: 'some_template_path.yml' }],
+            },
+          ],
+          policy_templates: [
+            {
+              inputs: [
+                {
+                  type: 'log',
+                  condition: "${host.platform} != 'linux'",
+                },
+              ],
+            },
+          ],
+        } as unknown) as PackageInfo,
```

Comment:
I'd prefer this avoid the assertion so we don't lose the type safety, but it's not a blocker